src/tqec/gallery/logical_cnot.py

```diff
@@ -0,0 +1,42 @@
+from tqec.computation.block_graph.graph import BlockGraph
+from tqec.computation.zx_graph import ZXGraph
+from tqec.position import Position3D
+
+
+def logical_cnot_block_graph(support_z_basis_observables: bool = True) -> BlockGraph:
+    logical_cnot_zx = _create_logical_cnot_zx_graph(support_z_basis_observables)
+    return logical_cnot_zx.to_block_graph(logical_cnot_zx.name)
+
+
+def _create_logical_cnot_zx_graph(support_z_basis_observables: bool = True) -> ZXGraph:
```

Comment:
It might be worth leaving the `ZXGraph` creation function public, I do not see the downside of having that function public.